src/Pretzel.Logic/Configuration.cs

```diff
@@ -17,8 +19,9 @@ public interface IConfiguration
         IDefaultsConfiguration Defaults { get; }
     }
 
-
-    internal sealed class Configuration : IConfiguration
+    [Shared]
+    [Export(typeof(IConfiguration))]
+    public sealed class Configuration : IConfiguration
```

Comment:
Do exported classes need to be public?

Reply:
Just tested it out, seams like it works. Doc's arn't specific about that. But importing constructors need to be public. Seam's extreme weird to me.